jekyll.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("colorator",             "~> 1.0")
   s.add_runtime_dependency("jekyll-sass-converter", "~> 1.0")
   s.add_runtime_dependency("jekyll-watch",          "~> 1.1")
-  s.add_runtime_dependency("kramdown",              "~> 1.3")
+  s.add_runtime_dependency("kramdown",              "~> 1.14", ">= 1.14.0")
```

Comment:
I wasn't sure if we should still allow versions < 1.14 when using rouge 1 but it would make things quite complicated and a bump should still work with rouge 1.